src/internal/serviceenv/config.go

```diff
@@ -96,6 +96,7 @@ type PachdSpecificConfiguration struct {
 	ImagePullSecrets           string `env:"IMAGE_PULL_SECRETS,default="`
 	MemoryRequest              string `env:"PACHD_MEMORY_REQUEST,default=1T"`
 	WorkerUsesRoot             bool   `env:"WORKER_USES_ROOT,default=false"`
+	WorkerVolumesMemoryBacked  bool   `env:"WORKER_VOLUMES_MEMORY_BACKED,default=false"`
```

Comment:
Why is this needed?

Reply:
Hey good catch, I initially put this here when I misread the JIRA's suggested solution. This should be removed.